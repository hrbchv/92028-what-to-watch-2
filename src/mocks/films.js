const films = [
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    imgSrc: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    genre: `Fantasy`,
    year: `2014`
  },
  {
    name: `Bohemian Rhapsody`,
    imgSrc: `img/bohemian-rhapsody.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    genre: `Drama`,
    year: `2015`
  },
  {
    name: `Macbeth`,
    imgSrc: `img/macbeth.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    genre: `Drama`,
    year: `2016`
  },
  {
    name: `Aviator`,
    imgSrc: `img/aviator.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    genre: `Drama`,
    year: `2017`
  },
  {
    name: `Some name`,
    imgSrc: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    genre: `Drama`,
    year: `2018`
  }
];

export default films;
