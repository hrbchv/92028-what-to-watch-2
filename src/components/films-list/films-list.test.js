import React from "react";
import renderer from "react-test-renderer";
import FilmsList from "./films-list.jsx";
import films from "../../mocks/films";

describe(`FilmsList`, () => {
  it(`should renders correctly`, () => {

    const tree = renderer
      .create(<FilmsList
        filmsCards={films}/>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
