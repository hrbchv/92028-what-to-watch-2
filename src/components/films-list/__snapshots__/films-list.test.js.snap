// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsList should renders correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        muted={true}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        preload="true"
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        muted={true}
        poster="img/bohemian-rhapsody.jpg"
        preload="true"
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        muted={true}
        poster="img/macbeth.jpg"
        preload="true"
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Macbeth
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        muted={true}
        poster="img/aviator.jpg"
        preload="true"
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Aviator
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        muted={true}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        preload="true"
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Some name
      </a>
    </h3>
  </article>
</div>
`;
